exec/session.go

```diff
@@ -383,7 +397,7 @@ func (s *Session) HandleDebug(handler *http.ServeMux) {
 // bigslice.Func.
 type Result struct {
 	bigslice.Slice
-	inv       bigslice.Invocation
+	inv       execInvocation
```

Comment:
This doesn't need to change, right?

Reply:
No; I changed it because I thought it might be confusing to see a `bigslice.Invocation` where the rest of the code in this package basically is using `execInvocation`. Strictly speaking, this actually only needs to hold the invocation index, as it's only used for transforming back into `invocationRef`s. So I think there are three options:

1. `inv bigslice.Invocation`
2. `inv execInvocation`
3. `invIndex uint64`

I'm now thinking that maybe `invIndex uint64` is the best choice, as it's the minimal thing necessary.

Reply:
I went ahead and just did what I suggested.